static/app/utils/replays/replayReader.tsx

```diff
@@ -588,7 +588,7 @@ export default class ReplayReader {
 
   getCustomFrames = memoize(() =>
     this._sortedBreadcrumbFrames.filter(
-      frame => !BreadcrumbCategories.includes(frame.category)
+      frame => frame.category && !BreadcrumbCategories.includes(frame.category)
```

Comment:
dles this need to be something like `'category' in frame && frame.category`?

Reply:
No, since `frame` is a `BreadcrumbFrame`, so it does have `frame.category`. However, for custom breadcrumbs, `category` is optional so `frame.category` could be `undefined`

Reply:
If it's the case that `BreadcrumbFrame` has `category, but cusom items do not have category, then it means that custom breadcrumbs are not of type `BreadcrumbFrame` and that will for sure mess up `_sortedBreadcrumbFrames`.